### ISSUE TITLE
"Further Reading" sections

### ISSUE THREAD

**Issue by p0llard** [achlipala/frap#49]
Something of a feature request, but perhaps it might be nice to have a "Further Reading" section at the end of each chapter with links to some relevant papers/readings?

I've got a list of papers somewhere which I found/read/skimmed when I took 822 which I'll add to this (when I've found it) on the off chance that it's useful.

**Comment by achlipala** [comment]
I agree that extra content would be nice, so I'm glad to have this issue here.  Perhaps I'll go after this one incrementally the next time I teach the class.
